tests/check_source_tests.py

```diff
@@ -70,6 +79,30 @@ def test_devel_project(self):
         self.assertReview(req_id, by_user=(self.bot_user, 'accepted'))
         self.assertReview(req_id, by_group=(REVIEW_TEAM, 'new'))
 
+    def test_source_maintainer(self):
+        """Declines the request when the 'required_maintainer' is not maintainer of the source project"""
```

Comment:
I'm missing a positive test. I have my suspicion....

Reply:
Good catch. It looks like `maintainers_get` only returns users, but not groups.

Reply:
Well, my assertion is not accurate: in case a group is a maintainer of the project, it returns the users within the group, not the group itself.